content/authorization/sign-in-with-livechat/index.mdx

```diff
@@ -66,268 +77,499 @@ The SDK is available in two formats: an NPM module and a CDN-hosted library.
+document.getElementById('login-button').onclick = (e) => {
+  if (e && e.preventDefault) {
+    e.preventDefault();
+  }
 
-> Native "Sign in with LiveChat" button
+  sdk.popup().authorize().then((authorizeData)=>{
+    const transaction = sdk.verify(authorizeData);
+    if (transaction != null) {
+      // authorization success
+      // authorizeData contains `accessToken` or `code`
+      // transaction contains state and optional code_verifier (code + PKCE)
+      console.log("User access token: " + transaction.accessToken)
+      document.getElementById('login-button').style.display = "none"
+    } else {
+      console.log("Redirect state don't matches previous one")
+    }
+  }).catch((e)=>{
+    console.error("Failed to authorize user", e)
+  })
+};
+```
 
-```html
-<div class="livechat-login-button"></div>
+</CodeSample>
+
+During the creation of the Sign in LiveChat button, you can use [LiveChat Designs](https://livechat.design/).
+
+<Warning>
+
+[**Don't forget to test the give consent step**](/monetization/app-review-process/#common-problems). Use `prompt: "consent"` to force the app to **ask you for access** to certain resources. It’s necessary for you to test the app as if you were a user who installs the app from Marketplace.
+
+</Warning>
+
+# SDK documentation
+
+## Classes
+
+### **AccountsSDK**
+
+The main instance of the SDK used to authorize users in LiveChat Accounts.
+
+#### **constructor(options = {})**
+
+<CodeSample path={'Example class creation'}>
+
+```js
+const instance = new AccountsSDK({
+  client_id: '<your_app_client_id>'
+});
+```
+
+</CodeSample>
+
+##### options object
+
+The constructor accepts an `options` object with the following properties:
+
+| Property                             | Required | Description                                                                                                                                                  |
+| ------------------------------------ | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `client_id`                          | yes      | Application **Client Id** (can be found in <a href="https://developers.livechat.com/console/" rel="noopener nofollow" target="_blank">Developer Console</a>) |
+| `prompt`                             | no       | Use `consent` to force the consent prompt in a popup and the redirect flows. ([See the warning.](#step-3-prepare-button-container)). Default: `''`           |
+| `response_type`                      | no       | OAuth response type. Possible values: `token` or `code`. Default: `token`                                                                                    |
+| `popup_flow`                         | no       | `auto` - close the popup when not required; `manual` - always show the popup; default: `auto`                                                                |
```

Comment:
A quick q about this part, "when not required". If you don't use popups to get user auth data, but you use e.g. Iframe instead, you will use `auto`?

Reply:
No, it's also affecting all other flows, but I don't know if it's by design

Reply:
So when you should use `auto` and when `manual`? Could you explain that in simple words? Because I don't know when it wouldn't be required.

Reply:
@ale-kacperczyk Updated :D